Sources/TuistDoc/SwiftDocController.swift

```diff
@@ -0,0 +1,59 @@
+import RxBlocking
+import TSCBasic
+import TuistCore
+import TuistSupport
+
+/// SwiftDoc controller protocol passed in initializers for dependency injection
+public protocol SwiftDocControlling {
```

Comment:
I think it'd be nice to add some `IntegrationTests`, so we know that the contract with `swift-doc` is not broken